src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.test.tsx

```diff
@@ -48,15 +49,17 @@ const createRandomPreviouslyViewedObject = (): PreviouslyViewedObject => ({
   genome_id: faker.random.word(),
   object_id: `${faker.random.word()}:gene:${faker.datatype.uuid()}`,
   object_type: 'gene',
-  label: faker.random.word()
+  label: faker.random.word(),
+  versioned_stable_id: faker.random.word()
 });
 
 const previouslyViewedObjects = [
   {
     genome_id: genomeId,
     object_id: geneObjectId,
     object_type: 'gene',
-    label: geneId
+    label: geneId,
+    versioned_stable_id: versionedStableId
```

Comment:
I don't know if there is anything that can be done about it, but I'll just notice here that the `versioned_stable_id` field does not quite fit the information model that we previously had for a `previouslyViewedObject`. Having one field named `label` while the other field named `versioned_stable_id` just feels wrong.